src/Uno.UI/Microsoft/UI/Xaml/Controls/InfoBadge/InfoBadge.properties.cs

```diff
@@ -1,53 +1,65 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
+// MUX reference InfoBadge.properties.cpp, tag winui3/release/1.4.2
 
-// MUX Reference InfoBadge.properties.cpp, tag winui3/release/1.4.2
-
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 using Windows.UI.Xaml;
 
-namespace Microsoft.UI.Xaml.Controls
+namespace Microsoft.UI.Xaml.Controls;
+
+public partial class InfoBadge
 {
-	public partial class InfoBadge
+	/// <summary>
+	/// Gets or sets the icon to be used in an InfoBadge.
+	/// </summary>
+	public IconSource IconSource
+	{
+		get => (IconSource)GetValue(IconSourceProperty);
+		set => SetValue(IconSourceProperty, value);
+	}
+
+	/// <summary>
+	/// Identifies the InfoBadge.IconSource dependency property.
+	/// </summary>
+	public static DependencyProperty IconSourceProperty { get; } =
+		DependencyProperty.Register(nameof(IconSource), typeof(IconSource), typeof(InfoBadge), new FrameworkPropertyMetadata(null, OnPropertyChanged));
+
+	private static void OnPropertyChanged(
+		DependencyObject sender,
+		DependencyPropertyChangedEventArgs args)
 	{
-		public IconSource IconSource
-		{
-			get => (IconSource)GetValue(IconSourceProperty);
-			set => SetValue(IconSourceProperty, value);
-		}
-
-		public static DependencyProperty IconSourceProperty { get; } =
-			DependencyProperty.Register(nameof(IconSource), typeof(IconSource), typeof(InfoBadge), new FrameworkPropertyMetadata(null, OnPropertyChanged));
-
-
-		public InfoBadgeTemplateSettings TemplateSettings
-		{
-			get => (InfoBadgeTemplateSettings)GetValue(TemplateSettingsProperty);
-			set => SetValue(TemplateSettingsProperty, value);
-		}
-
-		public static DependencyProperty TemplateSettingsProperty { get; } =
-			DependencyProperty.Register(nameof(TemplateSettings), typeof(InfoBadgeTemplateSettings), typeof(InfoBadge), new FrameworkPropertyMetadata(null, OnPropertyChanged));
-
-		private static void OnPropertyChanged(
-			DependencyObject sender,
-			DependencyPropertyChangedEventArgs args)
-		{
-			var owner = (InfoBadge)sender;
-			owner.OnPropertyChanged(args);
-		}
```

Comment:
Not needed?

Reply:
It is still there, just moved about due to whitespace change